package.json

```diff
@@ -35,7 +35,7 @@
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@joi/date": "^2.1.0",
     "@opensearch-project/opensearch": "^2.4.0",
-    "@uswds/uswds": "3.7.0",
+    "@uswds/uswds": "^3.7.1",
```

Comment:
I think we should leave this locked (to 3.7.1). I have zero faith in USWDS maintaining correct semver now

Reply:
I'm half with ya. I also no longer have faith in USDWS maintaining correct semver, but I'm a little disappointed this got through both manual and automated testing.

Reply:
Splitting hairs here, but it only got through manual testing _because_ it wasn't version locked

Reply:
But I do agree!